model/src/main/java/com/gdgistanbul/model/Model.kt

```diff
@@ -2,7 +2,10 @@ package com.gdgistanbul.model
 
 import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
+import com.sun.xml.internal.ws.developer.Serialization
+import java.io.Serializable
 
+@Serialization
```

Comment:
I think you don't need to use it. Moshi provides that

Reply:
i forgot this and fixed with seccond commit